lib/classes/squad.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:collection/collection.dart';
+import 'package:phone_factory/classes/squad_members/squad_member.dart';
+
+class Squad {
+  int _captain = 1;
+  static const int _initSquadMemberCount = 3;
```

Comment:
please, organize variables and methods in such order as 
- constructors
- constants
- variables
- getter/setters
- public methods
- private methods